src/spaces/openstreetmap.jl

```diff
@@ -112,7 +111,7 @@ function OpenStreetMapSpace(
     path::AbstractString;
     kwargs...
 )
-    m = graph_from_file(path; weight_type = :distance, kwargs...)
+    m = graph_from_file(path; weight_type=:distance, kwargs...)
```

Comment:
Why? No, please leave spaces between `=`.

Reply:
Thanks for pointing it out. Looks like the Julia extension is acting up, I'll just turn off the auto-formatter and do it manually.